extensions/amp-consent/0.1/consent-config.js

```diff
@@ -320,15 +322,21 @@ export class ConsentConfig {
  * Expand consent endpoint url
  * @param {!Element|!ShadowRoot} element
  * @param {string} url
+ * @param {Object<string, *>=} vars
  * @return {!Promise<string>}
  */
-export function expandConsentEndpointUrl(element, url) {
+export function expandConsentEndpointUrl(element, url, vars = {}) {
   return Services.urlReplacementsForDoc(element).expandUrlAsync(
     url,
     {
       'CLIENT_ID': getConsentCID(element),
+      'CONSENT_PAGE_VIEW_ID_64': () =>
+        getServicePromiseForDoc(element, 'consentStateManager').then(
```

Comment:
As discussed offline, please export a `getConsentStateManager` function from `consent-state-manager.js` and use it here, and in other places where we use `getServicePromiseForDoc(element, CONSENT_STATE_MANAGER)`
```suggestion
        getConsentStateManager(element).then(
```

Reply:
Sounds good. Thanks for the reminder